ci/prow/templates/prow_periodic_custom_job.yaml

```diff
@@ -4,7 +4,7 @@
   agent: kubernetes
   decorate: true
   [[indent_section 4 "decoration_config" .Base.DecorationConfig]]
-  [[indent_section 2 "extra_refs" .Base.ExtraRefs]]
+  [[indent_section 4 "extra_refs" .Base.ExtraRefs]]
```

Comment:
This is causing the indentation to be off.

Reply:
fixed